scripts/install-chef-suse.sh

```diff
@@ -534,25 +534,25 @@ for arch in $product_arches; do
         echo "Detected SUSE OpenStack Cloud Deps media."
     else
 
-        sp2mediasum="a1f512c3244c8e75da0f47d2d0752064"
+        sp3mediasum="59280f20e0bee15615b598a3ea3f027e"
```

Comment:
are we sure that those media sums don't change? I would assume we have to comment out the check for now while SP3 isn't released yet..